engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableAggregationTest.java

```diff
@@ -3824,4 +3832,54 @@ public void testPreserveEmptyNoKey() {
         TestCase.assertEquals(1, aggregated.size());
         assertTableEquals(expectedEmpty, aggregated);
     }
+
+    @Test
+    public void testSymbolTableBy() throws IOException {
+        diskBackedTestHarness((table) -> {
+            final Table result = table.aggBy(AggSum("Value"), "Symbol");
+            TableTools.showWithRowSet(result);
+
+            final long[] values = new long[result.intSize()];
+            final MutableInt pos = new MutableInt();
+            result.longColumnIterator("Value").forEachRemaining((long value) -> {
+                values[pos.getValue()] = value;
+                pos.increment();
+            });
+            assertArrayEquals(new long[] {0, 5, 17, 23}, values);
+        });
+    }
+
+    private void diskBackedTestHarness(Consumer<Table> testFunction) throws IOException {
+        final File directory = Files.createTempDirectory("QueryTableAggregationTest").toFile();
+
+        try {
+            final Table table = makeDiskTable(directory);
+
+            testFunction.accept(table);
+
+            table.close();
+        } finally {
+            FileUtils.deleteRecursively(directory);
+        }
+    }
+
+    @NotNull
+    private Table makeDiskTable(File directory) throws IOException {
```

Comment:
I think we should be testing with more regions. Should build multiple Parquet files with partially-overlapping sets of unique symbol values, created in different order (meaning the relative order of the first encounter of each symbol should be different across files).